index.html

```diff
@@ -1,19 +1,85 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
-
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
-
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Karma</title>
+        <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    </head>
+    <body>
+        <header class="header">
+            <img src="./img/karma-logo.svg" alt="Karma logo" class="logoImg"/>
+            <nav>
+                <ui class="navList">
+                    <li class="navItem">
+                        <a href="#" class="navLink"><span class="makeBlack">Meet Kamra</span></a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">How It works</a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">store</a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">Blog</a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">Help</a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">Login</a>
+                    </li>
+                </ui>
+            </nav>
+            <img class="hamburgerMenu" src="./img/menu-hamburger.svg" alt="menu bar"/>
+        </header>
+        <main>
```

Comment:
Good use of main 👍

Reply:
Thanks a lot Sam for reviewing my work,I will try to fix all the problems you have mentioned.